cluster/helm/chart/purser/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```

Comment:
Would this be better as "A Helm chart for the Purser extension to Kubernetes"?

Reply:
Oops, might have overlooked that!